torch_brain/nn/infinite_vocab_embedding.py

```diff
@@ -161,7 +161,7 @@ def extend_vocab(self, vocab: List[str], exist_ok=False):
         # update tokenizer
         self.vocab.update(
             OrderedDict(
-                zip(vocab, range(len(self.vocab), len(self.vocab) + len(vocab)))
+                zip(new_words, range(len(self.vocab), len(self.vocab) + len(new_words)))
             )
```

Comment:
This is a legit bug right? Want your confirmation... I feel we would've noticed it before if it was.

Reply:
good catch, yes, this is a bug. we have actually never used this because typically we do finetuning on new sessions, and not on a session we pretrained, on at least for POYO/POYO+.